app/queue_consumers/tagging_updater.rb

```diff
@@ -14,7 +14,13 @@ def process(message)
 private
   def process_content_item(content_item)
     return unless content_item['links']
-    artefact = find_artefact_by_base_path(content_item.fetch('base_path'))
+
+    # Panopticon pre-dates content items without base paths.
+    # We can ignore this content.
+    base_path = content_item.fetch('base_path')
```

Comment:
From the logs it looks like the key is there, but it's nil.

I'll write a test in a follow up PR.